packages/ethereum-contracts/contracts/utils/SuperfluidFrameworkDeploymentSteps.sol

```diff
@@ -41,7 +27,16 @@ import { IResolver } from "../interfaces/utils/IResolver.sol";
 /// @author Superfluid
 /// @notice A contract which splits framework deployment into steps.
```

Comment:
maybe a few notes on why this was needed in the first place, I don't think it's evident for people that have not experienced this issue.